presto-main/src/main/java/com/facebook/presto/execution/SqlQueryExecution.java

```diff
@@ -99,6 +104,8 @@
     private final List<Expression> parameters;
     private final SplitSchedulerStats schedulerStats;
 
+    private Set<ConnectorId> connectors;
```

Comment:
Is this just to carry result state from `doAnalyzeQuery()` to `start()`. If so, I'd just add this to PlanRoot, instead.